update: update the Test code using JUnit in JVM tutorial (#4158)
JetBrains/kotlin-web-site: docs/topics/jvm/jvm-test-using-junit.md

[//]: # (title: Test code using JUnit in JVM – tutorial)

This tutorial shows you how to write a simple unit test and run it with the Gradle build tool.

The example in the tutorial has the [`kotlin.test`](https://kotlinlang.org/api/latest/kotlin.test/index.html) library under the hood and runs the test using JUnit.

To get started, first download and install the latest version of [IntelliJ IDEA](https://www.jetbrains.com/idea/download/index.html).

## Add dependencies

> 1. Open a Kotlin project in IntelliJ IDEA. If you don't have a project, [create one](jvm-get-started.md#create-an-application).

Reviewer: ```suggestion
1. Open a Kotlin project in IntelliJ IDEA. If you don't have a project, [create one](jvm-get-started.md#create-a-project).
```

Reviewer: The instructions linked here tell the user to use IntelliJ for the build system, not Gradle. So maybe we shouldn't link to these steps? 🤔

Author: Yeah, you're right.
I changed the link to the IDEA's tutorial.

> 2. Open the `build.gradle(.kts)` file and add the following dependency to the Gradle configuration.

Reviewer: This dependency was already there when I created an IDEA project from scratch with the "hello world" code sample. So maybe we should change it to "check" if this dependency is present and if not, add it?

[//]: # (title: Test code using JUnit in JVM – tutorial)

This tutorial shows you how to write a simple unit test and run it with the Gradle build tool.

The example in the tutorial has the [`kotlin.test`](https://kotlinlang.org/api/latest/kotlin.test/index.html) library under the hood and runs the test using JUnit.

To get started, first download and install the latest version of [IntelliJ IDEA](https://www.jetbrains.com/idea/download/index.html).

## Add dependencies

1. Open a Kotlin project in IntelliJ IDEA. If you don't have a project,
   [create one](https://www.jetbrains.com/help/idea/create-your-first-kotlin-app.html#create-project).

> 2. Open the `build.gradle(.kts)` file and check that `testImplementation(kotlin("test"))` is present.

Reviewer: ```suggestion
2. Open the `build.gradle(.kts)` file and check that the `testImplementation`  dependency is present.
```
Visually, the wrapping of the instruction text looks a bit confusing because of alignment of the inline code bits. If you shorten `testImplementation(kotlin("test"))` so that it doesn't have Kotlin-specific syntax it looks fine. Totally up to you though!

4. Run the test again and check the result:

>    ![Check the test result. The test has been failed](test-failed.png)

Reviewer: ```suggestion
   ![Check the test result. The test has failed](test-failed.png)
```

   The test execution failed.

## What's next

Once you've finished your first test, you can:

> * Try to write another test using other [`kotlin.test`](https://kotlinlang.org/api/latest/kotlin.test/kotlin.test/) functions.

Reviewer: ```suggestion
* Write more tests using other [`kotlin.test`](https://kotlinlang.org/api/latest/kotlin.test/kotlin.test/) functions.
```
I think it sounds more positive without try :)
